### PR TITLE
Add TIME_ZONE parameter

### PR DESCRIPTION
**Pull request by denismosolov**

### CODE REVIEW

**Comment by eugulixes** [comment]
Do not use points at the end of commit messages.

File: bootstrap/15-locale.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
```
Comment (eugulixes) [inline bootstrap/15-locale.sh]: In this script you rely on the `TIME_ZONE` variable which for some reason can be undefined. Use

```bash
if ! check_if_variable_is_set TIME_ZONE; then
    >&2 echo "TIME_ZONE is not specified"
    exit 1
fi
```

to protect the script from unpredictable behaviour if the variable is unset.

File: bootstrap/15-locale.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Set timezone
+# https://bugs.launchpad.net/ubuntu/+source/tzdata/+bug/1554806
```
Comment (eugulixes) [inline bootstrap/15-locale.sh]: There are a comment and a related link, but still I think there is something missing. Probably we should provide a summary of the bug and after this provide the link for the ones who are interested in the details. What do you think?

File: bootstrap/15-locale.sh (hunk 1 above)
Comment (denismosolov) [reply bootstrap/15-locale.sh]: I've missed TIME_ZONE=${TIME_ZONE:='Etc/UTC'}

File: bootstrap/15-locale.sh (hunk 2 above)
Comment (denismosolov) [reply bootstrap/15-locale.sh]: I think that's fine. I don't want to overtext it.

File: bootstrap/15-locale.sh (hunk 1 above)
Comment (denismosolov) [reply bootstrap/15-locale.sh]: It assigns a default value here:  https://github.com/tolstoyevsky/pieman/blob/timezone/pieman.sh#L76
I cannot imagine a case when `TIME_ZONE` can be undefined.

**Pull request merged**